app/services/invoices/payments/retry_batch_service.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Invoices
+  module Payments
+    class RetryBatchService < BaseService
+      def initialize(organization_id:)
+        @organization_id = organization_id
+
+        super
+      end
+
+      def call
+        Invoices::Payments::RetryAllJob.perform_later(organization_id:, invoice_ids: invoices.ids)
+
+        result.invoices = invoices
+
+        result
+      end
+
+      def retry_all(invoice_ids)
```

Comment:
Just an idea: what about having this one name `call` and the first one `call_later`?

Just wondering if we should keep `call` name for the method that is performing the business logic.

That's maybe something we could discuss in the back team.